src/components/invoice/InvoiceComponent.tsx

```diff
@@ -142,4 +150,23 @@ export default class InvoiceComponent extends React.Component<Props, State> {
         return style.statusPaid;
     }
   }
+
+  private onPressDownload() {
+    const invoice = this.props.invoice;
```

Comment:
Consistency issue:
Use like you did in line 157:
`const { invoice } = this.props;`